src/parameterized_tendencies/radiation/RRTMGPInterface.jl

```diff
@@ -521,6 +521,16 @@ function RRTMGPModel(
                 end
                 lookups = (; lookups..., lookup_lw_cld)
             end
+            if radiation_mode.aerosol_radiation
+                local lookup_lw_aero
+                data_loader(joinpath("rrtmgp-aerosols-merra-lw.nc")) do ds
+                    lookup_lw_aero =
+                        RRTMGP.LookUpTables.LookUpAerosolMerra(ds, FT, DA)
+                end
```

Comment:
```suggestion
                lookup_lw_aero = data_loader(joinpath("rrtmgp-aerosols-merra-lw.nc")) do ds
                    RRTMGP.LookUpTables.LookUpAerosolMerra(ds, FT, DA)
                end
```

Reply:
Thanks! I was wondering why we need a local variable here.

Reply:
Oh, I don't know why it shows up in this PR. Maybe I need to rebase. This was added in #3156, following the pattern of lookup_lw_cld [here](https://github.com/CliMA/ClimaAtmos.jl/blob/e1aa08ce7a70cc96f507cd623aa25d80ee45b3fe/src/parameterized_tendencies/radiation/RRTMGPInterface.jl#L515). I will change it. Should I make similar changes to the cloud lookup tables?

Reply:
When I made this change (and similar changes to cloud lookup tables) I got NaN when running jobs interactively, and it was non-deterministic. I'll keep what we have for now as this PR is just renaming things (this change is not there anymore after rebasing), but I will open a new PR on removing the local variables.

Reply:
Ah, the non-deterministic result is from something else (about aerosols). I will fix that first, then change the local variables.